doc/libNX_X11/lcUniConv/README

```diff
@@ -0,0 +1,11 @@
+
+The files in this directory are taken from the libiconv-1.1 package.
```

Comment:
Don't you want to instead symlink `doc/_attic_/XOrg69_documentation_files/nx-X11_lib_X11_lcUniConv_README`?